cli/cmd/sudo/project/search.go

```diff
@@ -50,6 +52,7 @@ func SearchCmd(ch *cmdutil.Helper) *cobra.Command {
 			return nil
 		},
 	}
+	searchCmd.Flags().StringArrayVar(&tags, "tag", []string{}, "Tags to filter projects by")
```

Comment:
Not sure, but I think this should be `StringSliceVar` to support multiple flags mapped to one array? Like `--tag tag1 --tag tag2`

Reply:
Looks like both are working fine, `StringSliceVar` make more sense here, updating this 👍